libs/components/ag-grid/src/lib/modules/ag-grid/top-horizontal-scroll/top-horizontal-scroll.service.ts

```diff
@@ -0,0 +1,76 @@
+import { ElementRef, Injectable, OnDestroy } from '@angular/core';
+import { MutationObserverService } from '@skyux/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TopHorizontalScrollService implements OnDestroy {
```

Comment:
This class name should have a `Sky` prefix. My suggestion would be `SkyAgGridTopScrollService` (I don't know that we need the word "horizontal" anywhere we refer to this concept).

Reply:
Renamed.